src/components/HeroSection/HeroSection.css

```diff
@@ -0,0 +1,69 @@
+.hero-wrapper{
+    grid-area: hero-section;
+    min-height: calc(90vh);
+    padding: 1rem;
+    max-width: 100%;
+    gap:0.4rem;
+}
+.hero-image{
+    margin: 0;
+    max-height: 50vh;
+}
+.responsive-img{
+    max-inline-size: 100%;
+    block-size: auto;
+    font-size: inherit;
+    display: block;
+    max-width: 100%;
+}
+.hero-heading{
+    max-width: max-content;
+    text-align: center;
+    color: var(--bg-color);
+    font-family:"Roboto-slab",serif;
+}
+
+.hero-links-wrapper{
+    margin-top: 1.5rem;
+    gap: 1rem;
+    max-width: 100%;
+}
+
+.hero-links-wrapper a{
+    text-decoration: none;
+}
+
+.hero-btn{
+    display: inline-block;
+    vertical-align: middle;
+    width: max-content;
+    min-width: 80px;
+    height: max-content;
+    border: 2px solid #0d1c29;
+    border-radius: var(--border-rd-sm);
+    color: #fafafa;
+    background-color: #0d1c29;
+    text-align: center;
+    font-weight: 500;
+    cursor: pointer;
+    display: flex;
```

Comment:
since you have created utilities  for flex styles in index.css you can use them here instead  of writing again